fipy/variables/operatorVariable.py

```diff
@@ -287,6 +291,11 @@ def _py3kInstructions(self, op, style, argDict, id, freshen):
                     stack.append(self._unop[ins.opcode] + '(' + stack.pop() + ')')
                 elif ins.opcode in self._binop:
                     stack.append(stack.pop(-2) + " " + self._binop[ins.opcode] + " " + stack.pop())
+                elif ins.opname == 'UNARY_NEGATIVE':
```

Comment:
A lot of this code is of the form:

```
 if dis.opname[bytecode] == 'UNARY_CONVERT':
                    stack.append("`" + stack.pop() + "`")
```
It could be cleaned up by having a dictionary that relates `UNARY_CONVERTER` to what's inside the `stack.append`. You would then just have one line of

```
stack.append(relation_dict.get(disopname[bytecode], otherfunc()))
```
and use `get` to call another function for the edge cases.